### ISSUE TITLE
Error documentaion of the function instrumentation

### ISSUE THREAD

**Issue by mistoan** [verificarlo/verificarlo#205]
I believe there is an error in the description of the function instrumentation section.
Currently it is:
```
You can produce a log file to summarize the vprec backend activity during the execution by giving the name of the file with the --prec-output-file parameter.
```
It should be:
```
You can produce a log file to summarize the vprec backend activity during the execution by giving the name of the file with the --prec-log-file parameter.
```

**Comment by pablooliveira** [comment]
Thanks for the patch. Could you please do a PR fixing this ?

**Comment by mistoan** [comment]
I've just noticed that the function instrumentation section isn't present in the overview of the readme file either.
Will do a pull request to fix it.

**Comment by mistoan** [comment]
I've corrected these issues in pull request #207.
